packages/ssh-tunnel/src/index.ts

```diff
@@ -70,57 +73,16 @@ export class SshTunnel extends EventEmitter {
 
     this.sshClient = new SshClient();
 
+    this.sshClient.on('close', () => {
+      debug('sshClient closed');
+      this.connected = false;
+    });
```

Comment:
Is there a race condition here? When the ssh conneciton is in the process of timing out, `.connected` is still `true`?

Reply:
I slapped a try/catch around where it forwards to retry now and wrote more tests.